integrators/cohen_hubbard_oesterwinter.hpp

```diff
@@ -0,0 +1,34 @@
+﻿
+#pragma once
+
+#include "numerics/fixed_arrays.hpp"
+
+namespace principia {
+namespace integrators {
+namespace internal_cohen_hubbard_oesterwinter {
+
+using numerics::FixedVector;
+
+// Definition of a Cohen-Hubbard-Oesterwinter formula.  |order_| is the order of
```

Comment:
Cite the paper, document any notational differences from the paper (I think we're off-by-one compared to it?)

Reply:
Done.